src/libcore/num/mod.rs

```diff
@@ -1318,6 +1460,40 @@ $EndFeature, "
             }
         }
 
+
+        doc_comment! {
+            concat!("Calculates the modulo of Euclidean divsion `self.mod_euc(rhs)`.
```

Comment:
Nit: I would call this "the remainder" or "the modulus", as "modulo" is the operation, rather than the result.

Reply:
Changing to "remainder" since "modulus" is another word for "divisor".